runtime/android/sample/src/org/xwalk/core/sample/ExtensionEcho.java

```diff
@@ -9,7 +9,7 @@
 public class ExtensionEcho extends XWalkExtension {
 
     public ExtensionEcho() {
-        super("echo",
+        super("echo_java",
```

Comment:
Sorry, I did not make it clear. The file name should be like `foo_bar`. The Java/JavaScript variable name should be like `fooBar`

Please refer to the chromium coding style page at https://www.chromium.org/developers/coding-style